app/src/org/odk/collect/android/tasks/FormLoaderTask.java

```diff
@@ -133,7 +136,7 @@ protected FECWrapper doInBackground(Uri... form) {
                 fis = new FileInputStream(formXml);
                 XFormParser.registerHandler("intent", new IntentExtensionParser());
                 XFormParser.registerStructuredAction("pollsensor", new PollSensorExtensionParser());
-                fd = XFormUtils.getFormFromInputStream(fis);
+                fd = XFormUtils.getFormFromInputStream(fis, new ImageRestrictionExtensionParser());
```

Comment:
I'm curious why you didn't follow the existing pattern of registering the extension parser statically, like ```registerStrucuturedAction```. Both approaches seem fine, but it feels weird to mix them because they require different init procedures, like having to pass the extension parser into ```getFormFromInputStream``` instead of calling a static registration method.